packages/react/label/src/Label.stories.tsx

```diff
@@ -28,6 +29,56 @@ export const WithHtmlFor = () => (
   </>
 );
 
+export const WithControl = () => {
+  return (
+    <>
+      <h1>Wrapping control</h1>
+      <Label>
+        <Checkbox.Root className={controlClass}>
+          <Checkbox.Indicator className={indicatorClass} />
+        </Checkbox.Root>{' '}
+        Label
+      </Label>
+
+      <h1>Referencing control</h1>
+      <Checkbox.Root id="control" className={controlClass}>
+        <Checkbox.Indicator className={indicatorClass} />{' '}
+      </Checkbox.Root>
+      <Label htmlFor="control">Label</Label>
+
+      <h1>Wrapping control - stopping propagation</h1>
+      <Label>
+        <Checkbox.Root className={controlClass}>
+          <Checkbox.Indicator className={indicatorClass} />
+        </Checkbox.Root>{' '}
+        Label.{' '}
+        <span
+          aria-hidden
```

Comment:
Our a11y eslint rules moan about non-interactive elements having `onClick` handlers without a keyboard handler. Adding this `aria-hidden` silenced it.